statsdaemon.go

```diff
@@ -289,12 +289,12 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 
 		switch {
 		case hasChanged:
-			fmt.Fprintf(buffer, "%s %d %d\n", bucket, currentValue, now)
+			fmt.Fprintf(buffer, "%s %f %d\n", bucket, currentValue, now)
 			lastGaugeValue[bucket] = currentValue
 			gauges[bucket] = math.MaxUint64
```

Comment:
That's a interesting point - in fact, MaxUint64 is one of the values that float64 can't exactly represent. (At that magnitude/exponent, the gap between successive values of float64 is 2.0)

```
[pierce@plo-pro ~]$ tail -n5 testmax.go 
func main() {
	var bigint uint64 = math.MaxUint64
	var bigdbl float64 = float64(bigint)
	fmt.Printf("%d\n%f\n", bigint, bigdbl)
}
[pierce@plo-pro ~]$ ./testmax 
18446744073709551615
18446744073709551616.000000
```
This works because it's converted to float where it's used and it's consistent.

But it's probably still a good idea to change this because, if someone is debugging code around here in the future, and printing or inspecting values, it's confusing.

Maybe this is even a good time to refactor `guages` + `lastGagueValue` into just a slice of
```
type GaugeInfo struct {
    value float64
    hasChanged bool
}
```